store/postgres/src/store.rs

```diff
@@ -657,6 +790,47 @@ impl Store {
         }
         Ok(())
     }
+
+    fn add_subscription(&self, subscription: Subscription) {
+        let subscriptions = self.subscriptions.clone();
+
+        // Generate a new (unique) UUID; we're looping just to be sure we avoid collisions
+        let mut id = Uuid::new_v4().to_string();
+        while subscriptions.read().unwrap().contains_key(&id) {
+            id = Uuid::new_v4().to_string();
+        }
+
+        {
+            let (entities, kind) = match &subscription {
+                Subscription::Changes { entities, .. } => (entities, "Changes"),
+                Subscription::Ticks { entities, .. } => (entities, "Ticks"),
+            };
+            debug!(self.logger, "Subscribe";
+               "id" => &id,
+               "kind" => kind,
+               "entities" => format!("{:?}", entities));
+        }
+
+        // Add the new subscription
+        let mut subscriptions = subscriptions.write().unwrap();
+        subscriptions.insert(id, subscription);
+    }
+
+    fn emit_block_tick(
+        &self,
+        conn: &PgConnection,
+        event_source: EventSource,
+    ) -> Result<(), StoreError> {
+        // Only emit block ticks for actual blocks
+        if event_source != EventSource::None {
+            trace!(self.logger, "Emit block tick"; "block" => event_source.to_string());
+            let v = serde_json::to_value(BlockTick {
+                block: event_source,
+            })?;
+            select(pg_notify("entity_changes", v.to_string())).execute(conn)?;
+        }
+        return Ok(());
```

Comment:
Drop the return and the semicolon. The last expression becomes the return value (similar to Ruby) unless you add a semicolon at the end of it.